packages/babel-plugin/__fixtures__/colors.stylex.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type { VarGroup } from '@stylexjs/stylex';
+
+import * as stylex from '@stylexjs/stylex';
+
+opaque type SemanticColorsTheme: 'theme' = 'theme';
+
+const SemanticColors = {
+  '--accent': null as null | string,
```

Comment:
Weird pattern to have all these values as `null`?

Reply:
This is a workaround for a performance pitfall in browsers, but not needed for this test.